Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -114,6 +120,7 @@ def StyleError.errorCode (err : StyleError) : String := match err with
   | StyleError.broadImport _ => "ERR_IMP"
   | StyleError.lineLength _ => "ERR_LIN"
   | StyleError.fileTooLong _ _ _ => "ERR_NUM_LIN"
+  | StyleError.newUnicode _ => "NO_ERR_CODE_AVAILABLE"
```

Comment:
You can just make one up, perhaps "ERR_UNICODE".

Can you also update the logic for `parse?_errorContext` below, please? (You can follow how e.g. the `ERR_AUT` code is treated.)

Reply:
I put the "test" I made for this `parse?_errorContext` into ` test/LintStyle.lean`.